app/src/test/java/org/oppia/android/app/topic/questionplayer/QuestionPlayerActivityLocalTest.kt

```diff
@@ -117,11 +111,11 @@ class QuestionPlayerActivityLocalTest {
       testCoroutineDispatchers.runCurrent()
       onView(withId(R.id.question_recycler_view)).check(matches(isDisplayed()))
 
-      // Submit correct answer
+      // Submit correct answer & wait until animation starts
```

Comment:
Why don't the other tests in this PR need to advance the time? Also, it seems like the test below is advancing time but not this one despite the comment--is that correct?

Reply:
I was actually mixing this up with the end of exploration confetti that has a delay where we would need to adjust time. We don't need to do any waiting here, I was simultaneously playing around with the animation times and these tests and mixed them all up.

Removing the waits